dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -52,6 +52,15 @@ def feed_ndarray(dali_tensor, arr, cuda_stream = None):
                     In most cases, using the default internal user stream or stream 0
                     is expected.
     """
+    if isinstance(dali_tensor, (TensorListCPU, TensorListGPU)):
```

Comment:
We cannot do such thing for PaddlePaddle as `feed_ndarray` there accepts raw pointer. We cannot accept tensor itself there as we cannot extract pointer itself to it without providing placement and type, so we would need to extend `feed_ndarray` signature and move allocation of data there as well (setting shape and placement does that). See [the API](https://github.com/PaddlePaddle/Paddle/blob/572bad8a66ea9650d753911af8383785ed671af9/paddle/fluid/pybind/pybind.cc#L638).